Formula/l/logdy.rb

```diff
@@ -0,0 +1,19 @@
+class Logdy < Formula
+  desc "Web based real-time log viewer"
+  homepage "https://logdy.dev"
+  # uncomment with the next release
```

Comment:
is this what you want?
```suggestion
  # TODO: switch to github source tarball
```